plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergSystemTables.java

```diff
@@ -236,5 +245,80 @@ public void testFilesTable()
+                            "('null_value_counts', 'map(integer, bigint)', '', '')," +
+                            "('nan_value_counts', 'map(integer, bigint)', '', '')," +
+                            "('lower_bounds', 'map(integer, varchar)', '', '')," +
+                            "('upper_bounds', 'map(integer, varchar)', '', '')," +
+                            "('key_metadata', 'varbinary', '', '')," +
+                            "('split_offsets', 'array(bigint)', '', '')," +
+                            "('equality_ids', 'array(integer)', '', '')");
+
+            assertQuerySucceeds("SELECT * FROM \"" + table.getName() + "$all_files\"");
+
+            MaterializedResult filesAfterSingleRow = computeActual("SELECT file_format, record_count, lower_bounds, upper_bounds FROM \"" + table.getName() + "$files\"");
+            MaterializedResult allFilesAfterSingleRow = computeActual("SELECT file_format, record_count, lower_bounds, upper_bounds FROM \"" + table.getName() + "$all_files\"");
+            assertEquals(filesAfterSingleRow, allFilesAfterSingleRow);
+
+            MaterializedResult expectedFilesStatsAfterSingleRow =
+                    MaterializedResult.resultBuilder(getSession(), VARCHAR, BIGINT, new MapType(INTEGER, VARCHAR, new TypeOperators()), new MapType(IntegerType.INTEGER, VARCHAR, new TypeOperators()))
+                            .row("ORC", 1L, Map.of(1, "1", 2, "a"), Map.of(1, "1", 2, "a"))
+                            .build();
+
+            assertEquals(filesAfterSingleRow, expectedFilesStatsAfterSingleRow);
+
+            Long snapshotAfterSingleRow = (Long) computeScalar("SELECT snapshot_id FROM \"" + table.getName() + "$snapshots\" ORDER BY committed_at DESC LIMIT 1");
+
+            assertUpdate("INSERT INTO " + table.getName() + " VALUES (2, 'b')", 1);
+
+            MaterializedResult filesAfterTwoRows = computeActual("SELECT file_format, record_count, lower_bounds, upper_bounds FROM \"" + table.getName() + "$files\" order by file_path");
+            MaterializedResult allFilesAfterTwoRows = computeActual("SELECT file_format, record_count, lower_bounds, upper_bounds FROM \"" + table.getName() + "$all_files\" order by file_path");
+
+            MaterializedResult expectedAllFilesStatsAfterTwoRows =
+                    MaterializedResult.resultBuilder(getSession(), VARCHAR, BIGINT, new MapType(INTEGER, VARCHAR, new TypeOperators()), new MapType(IntegerType.INTEGER, VARCHAR, new TypeOperators()))
+                            .row("ORC", 1L, Map.of(1, "1", 2, "a"), Map.of(1, "1", 2, "a"))
+                            .row("ORC", 1L, Map.of(1, "2", 2, "b"), Map.of(1, "2", 2, "b"))
+                            .build();
+            assertEquals(filesAfterTwoRows, allFilesAfterTwoRows);
+
+            assertEquals(filesAfterTwoRows, expectedAllFilesStatsAfterTwoRows);
+
+            assertUpdate("CALL iceberg.system.rollback_to_snapshot('tpch', '" + table.getName() + "' , " + snapshotAfterSingleRow + ")");
```

Comment:
nit: Remove a redundant space.
```suggestion
            assertUpdate("CALL iceberg.system.rollback_to_snapshot('tpch', '" + table.getName() + "', " + snapshotAfterSingleRow + ")");
```

Reply:
done